### DISCUSSION TITLE
neomutt truecolor

### DISCUSSION THREAD

**Discussion by b1nhack** [kovidgoyal/kitty#7521] [General]
Neomutt supports true color, but it requires the terminal type to be set to xterm-direct. In theory, Kitty also supports true color, and you can force true color in Neomutt. However, when using xterm-kitty, Neomutt exhibits color glitches. I would like to know if there is any way to use Neomutt's true color properly.

**Comment by kovidgoyal** [comment]
No idea, you would need to ask the neomutt developers.
